starter_code/app.js

```diff
@@ -10,9 +10,30 @@ app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(path.join(__dirname, 'public')));
 
+hbs.registerPartials(__dirname + '/views/partials');
+
 app.get('/', (req, res, next) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res, next) => {
+    punkAPI.getBeers()
+    .then(beers => {
+      res.render('beers', {beers});
+    })
+    .catch(error => {
+      console.log(error);
+    })   
+});
+
+app.get('\random-beers', (req, res, next) =>{
```

Comment:
Whoops! You used a backslash here, this should be `/random-beers`. This is why the route handler is not matching and you are getting a 404